src/main/java/carpet/logging/logHelpers/PathfindingVisualizer.java

```diff
@@ -2,37 +2,29 @@
 
 import carpet.helpers.ParticleDisplay;
 import carpet.logging.LoggerRegistry;
+import carpet.logging.TypeLogger;
 import net.minecraft.server.level.ServerPlayer;
 import net.minecraft.world.entity.Entity;
 import net.minecraft.world.phys.Vec3;
 
 public class PathfindingVisualizer
 {
+    @SuppressWarnings("unchecked")
```

Comment:
Ugh, big block for suppressing... Isn't there a more specific place to put it? I know on local declarations you can put it, not sure if in calls?

Reply:
It's not worth changing, since this method is basically just the log() and I would need to separate everything individually.